Configuration/DataProcessing/python/Impl/ppEra_Run2_2017_trackingOnly.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+"""
+_ppEra_Run2_2017_trackingOnly
+
+Scenario supporting proton collisions and tracking only reconstruction for HP beamspot
+
+"""
+
+import os
+import sys
+
+from Configuration.DataProcessing.Reco import Reco
+import FWCore.ParameterSet.Config as cms
+from Configuration.Eras.Era_Run2_2017_cff import Run2_2017
+
+from Configuration.DataProcessing.Impl.pp import pp
+
+class ppEra_Run2_2017_trackingOnly(pp):
+    def __init__(self):
+        pp.__init__(self)
+        # tracking only RECO is sufficient, to run high performance BS at PCL;
+        # some dedicated customization are required, though: customisePostEra_Run2_2017_trackingOnly
+        self.recoSeq=':reconstruction_trackingOnly'
+        self.cbSc=self.__class__.__name__
```

Comment:
it looks like overloading the meaning of ```pp``` to define both the ConfigBuilder scenario and to chose if the EI step should be included is no longer working.
The solution could be to add a flag ```addEI``` to the Reco class and set it to true in pp.py (and other relevant places where cbSc is set to "pp" already) and then set it to false in trackingOnly.py
A bit simpler but less robust is to add in Reco.py a check that recoSeq is not equal to ```reconstruction_trackingOnly ``` to still add the EI step